spec/unit/spotlight/dor/indexer_spec.rb

```diff
@@ -25,27 +28,211 @@
       it "does not parse series number from '#{example}'" do
-        ng_mods = Nokogiri::XML("#{mods_start}#{example}#{mods_end}")
+        ng_mods = Nokogiri::XML("#{mods_loc_phys_loc_start}#{example}#{mods_loc_phys_loc_end}")
         allow(r).to receive(:mods).and_return(ng_mods)
         subject.send(:add_series, sdb, solr_doc)
-        expect(solr_doc).not_to include 'series_ssim'
+        expect(solr_doc['series_ssi']).to be_falsey
       end
     end
   end # add_series
+
+  describe "#add_box" do
+    parsable_exemplars_based_on_actual_data = [
+      # feigenbaum
+      'Call Number: SC0340, Accession 2005-101, Box : 42, Folder: 1',
+      'Call Number: SC0340, Accession 2005-101, Box: 42, Folder: 3',
+      'Call Number: SC0340, Accession: 1986-052, Box 42 Folder 38',
+      'Call Number: SC0340, Accession: 2005-101, Box : 42, Folder: 20',
+      'Call Number: SC0340, Accession: 1986-052, Box: 42, Folder: 1',
+      'SC0340, 1986-052, Box 42',
+      'SC0340, Accession 1986-052, Box 42',
+      # shpc
+      'Series Biographical Photographs | Box 42 | Folder Abbot, Nathan',
+      'Series General Photographs | Box 42 | Folder Administration building--Outer Quad',
+      # menuez
+      'MSS Photo 451, Series 1, Box 42, Folder 11, Sleeve 32-11-2, Frame B32-F11-S2-6',
+      'Series 1, Box 42, Folder 8',
+      # fuller
+      'Collection: M1090 , Series: 4 , Box: 42 , Folder: 10',
+      # hummel (actually in <relatedItem><location><physicalLocation>)
+      'Box 42 | Folder 3',
+      'Flat-box 42 | Volume 1'
+    ]
+    parsable_exemplars_based_on_actual_data.each do |example|
+      it "parses box number from physLoc '#{example}'" do
+        ng_mods = Nokogiri::XML("#{mods_loc_phys_loc_start}#{example}#{mods_loc_phys_loc_end}")
+        allow(r).to receive(:mods).and_return(ng_mods)
+        subject.send(:add_box, sdb, solr_doc)
+        expect(solr_doc['box_ssi']).to eq '42'
+      end
+      it "parses box number from relItem physLoc '#{example}'" do
+        ng_mods = Nokogiri::XML("#{mods_rel_item_loc_phys_loc_start}#{example}#{mods_rel_item_loc_phys_loc_end}")
+        allow(r).to receive(:mods).and_return(ng_mods)
+        subject.send(:add_box, sdb, solr_doc)
+        expect(solr_doc['box_ssi']).to eq '42'
+      end
+    end
+    boxes_with_letters = [
+      # feigenbaum
+      'Call Number: SC0340, Accession 2005-101, Box: 42A, Folder: 24',
+      'Call Number: SC0340, Accession: 1986-052, Box: 42A, Folder: 59'
+    ]
+    boxes_with_letters.each do |example|
+      it "parses box number from '#{example}'" do
+        ng_mods = Nokogiri::XML("#{mods_loc_phys_loc_start}#{example}#{mods_loc_phys_loc_end}")
+        allow(r).to receive(:mods).and_return(ng_mods)
+        subject.send(:add_box, sdb, solr_doc)
+        expect(solr_doc['box_ssi']).to eq '42A'
+      end
+    end
+
+    unparsable_exemplars_based_on_actual_data = [
+      # feigenbaum
+      'Call Number: SC0340, Accession 2005-101',
```

Comment:
dropping 42A  boxes on the floor?   I think we'll have plenty of them.  Or did you mean ones without any box?   Does anyone ever check the indexing logging for spotlight?  I'm inclined to think no one does, since it is choked with DEBUG level messages, and probably has been for some time.